libs/ui/src/export_logs_modal.test.tsx

```diff
@@ -168,7 +169,9 @@ test('successful save raw log flow', async () => {
   userEvent.click(screen.getByText('Save'));
   await screen.findByText(/Saving Logs/);
   expect(mockKiosk.readFile).toHaveBeenCalled();
-  jest.advanceTimersByTime(2001);
+  act(() => {
+    jest.advanceTimersByTime(2001);
+  });
```

Comment:
In some places, the upgrade forces us to have proper `act` warnings. In others, previously necessary `act` wraps are no longer necessary. In general, I wouldn't say it's become more or less important, but React & RTL seem to be more consistent and logical about where they're required.